### PR TITLE
Update nginx conf for better performance

### PR DESCRIPTION
**Pull request by phoopw**
- add cache for static files
- add text compression for js files
- fastcgi cache config update

### CODE REVIEW

File: prod-ansible-role/roles/nginx/templates/vhost_orig.conf.j2 [hunk 1]
```diff
@@ -120,9 +122,8 @@ server {
     location ~* \.(jpg|jpeg|gif|png|css|js|ico|svg|webp)$ {
         access_log        off;
         log_not_found     off;
-        # Add 8 days expires to production
-        # We won't add cache for dev
-        add_header Cache-Control "s-maxage=86400";
+        # Add 7 days expires to production
```
Comment (biplobice) [inline prod-ansible-role/roles/nginx/templates/vhost_orig.conf.j2]: @phoopw It's 365 days, isn't it?

File: prod-ansible-role/roles/nginx/templates/vhost_orig_basicauth.conf.j2 [hunk 2]
```diff
@@ -130,9 +132,8 @@ server {
     location ~* \.(jpg|jpeg|gif|png|css|js|ico|svg|webp)$ {
         access_log        off;
         log_not_found     off;
-        # Add 8 days expires to production
-        # We won't add cache for dev
-        # add_header Cache-Control "s-maxage=86400";
+        # Add 7 days expires to production
```
Comment (biplobice) [inline prod-ansible-role/roles/nginx/templates/vhost_orig_basicauth.conf.j2]: Same as above.

**Review by biplobice — CHANGES_REQUESTED** [review]

File: prod-ansible-role/roles/nginx/templates/vhost_orig_basicauth.conf.j2 (hunk 2 above)
Comment (phoopw) [reply prod-ansible-role/roles/nginx/templates/vhost_orig_basicauth.conf.j2]: @biplobice I've changed the comment.

**Review by biplobice — APPROVED** [review]
@hissy LGTM.

**Comment by hissy** [comment]
@phoopw @biplobice 

Thanks!

**Pull request merged**